OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/dialog/EditSubkeyDialogFragment.java

```diff
@@ -95,6 +99,25 @@ public void onClick(DialogInterface dialog, int id) {
         return builder.show();
     }
 
+    private void showAlertDialog() {
+        CustomAlertDialogBuilder stripAlertDialog = new CustomAlertDialogBuilder(getActivity());
+        stripAlertDialog.setTitle(getResources().getString(R.string.title_alert_strip)).
```

Comment:
doesn't `setTitle` have a version which uses the resource id directly, just like `setMessage`?